src/HID-APIs/SurfaceDialAPI.hpp

```diff
@@ -81,10 +106,44 @@ void SurfaceDialAPI::release(void)
 void SurfaceDialAPI::releaseAll(void)
 {
 	_button = false;
-	rotate(0);
+	update();
+}
+
+void SurfaceDialAPI::onScreen(bool s)
+{
+	_onScreen = s;
+	update();
+}
+
+void SurfaceDialAPI::update()
```

Comment:
Not very intuitive what the difference between posrot and update is. It turns out the difference is in the rotation variable.